src/modules/extendedCallWindow/docs/de_DE.md

```diff
@@ -183,6 +183,12 @@ Du würdest dir gerne eigene Texte hinter oder vor den Einsatznamen schreiben? K
 
 ## Alarm-Icons
 
-Als Letztes kommen noch die Alarm-Icons, mit welchem auf dem Alarmieren-Button extra Symbole erscheinen, in Abhängigkeit davon, welche Moule aktiviert sind. Es stehen alle kostenlosen Icons von <a :href="$theme.variables.fontAwesomeIconSearchLink" target="_blank">FontAwesome</a> zur Verfügung.
+Auswählbare Alarm-Icons, welche auf dem Alarmieren-Button als extra Symbole erscheinen, in Abhängigkeit davon, welche Moule aktiviert sind. Es stehen alle kostenlosen Icons von <a :href="$theme.variables.fontAwesomeIconSearchLink" target="_blank">FontAwesome</a> zur Verfügung.
```

Comment:
War schon vorher drin, aber "in Abhängigkeit davon, welche Moule aktiviert sind." ist falsch.
Korrekt wäre "in Abhängigkeit davon, welche Fahrzeuge ausgewählt sind."